src/i18n/translations/en.ts

```diff
@@ -650,4 +650,10 @@ export const messages: MessagesKey = {
   mapLoadingStopsDataText: 'Loading stops...',
   locateStopPoint: 'Locate',
   locateStopPointTooltip: 'Locate on map',
+  exportsDownloadUnsupportedBrowserTitle: 'Browser Not Supported',
+  exportsDownloadUnsupportedBrowserMessage:
+    'Sorry, your browser does not support file downloads.',
+  exportsDownloadErrorTitle: 'Download Failed',
```

Comment:
Swedish: Nedladdning misslyckades

Reply:
Finnish: _Lataus epäonnistui_

Literally the same :)